platform/commonUI/browse/res/templates/browse.html

```diff
@@ -24,7 +24,7 @@
     <mct-include key="'topbar-browse'"></mct-include>
     <div class="abs holder holder-main browse-area s-browse-area browse-wrapper"
          ng-controller="PaneController as modelPaneTree"
-         ng-class="modelPaneTree.visible() ? 'pane-tree-showing' : 'pane-tree-hidden'">
+         ng-class="modelPaneTree.visible() ? 'pane-tree-showing' : 'pane-tree-hidden'" hideParameter="hideTree">
```

Comment:
Html attributes are case insensitive, we stick to lowercase.  In order to get this to camelCase in $attrs as `hideParameter`, you should change this to `hide-parameter` in the HTML.